scripts/HO-example.jl

```diff
@@ -0,0 +1,112 @@
+# # Example 5: Harmonic oscillator
+
+# This is an example calculation of the harmonic oscillator with
+# contact interactions that are treated with first-order perturbation
+# theory.
+
+# A runnable script for this example is located
+# [here](https://github.com/joachimbrand/Rimu.jl/blob/develop/scripts/HO-example.jl).
+# Run it with `julia HO-example.jl`.
+
+# Firstly, we load all needed modules.
+using Rimu
+using DataFrames
+using LinearAlgebra
+
+# First we define the system size for two particles in a 2D harmonic oscillator
+# allowing `M` levels above the ground state in each dimension.
+N = 2
+D = 2
+M = 3
+S = ntuple(_ -> M + 1, D)
+P = prod(S)
+
+# The tuple `S` defines a grid of harmonic oscillator states in a Cartesian
+# basis (`P` is the total number of single-particle states), in this case 
+# ``n_x,n_y=0,1,\ldots,M``.
+# In `Rimu` the ``N``-particle states are still stored as Fock states.
+# Use the utility function [`fock_to_cart`](@ref) to convert a Fock address to 
+# human-readable Cartesian quantum numbers for inspection.
+addr = BoseFS(P, M + 1 => N)
+fock_to_cart(addr, S)
+# The output shows that all ``N`` particles are in single-particle state ``n_x=M, n_y=0``.
```

Comment:
It would be instructive to also show how `addr` is printed (as a Fock state) in order to appreciate the difference.